src/UI/Component/Button/Split/Standard.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Button\Split;
+
+/**
+ * This describes a standard split button.
+ */
+interface Standard extends Split {
+	/**
+	 * Get the array containing the actions and labels of the split button
+	 *
+	 * @return	array (string|string)[]. Array containing keys as label and values as actions.
+	 */
+	public function getActionsAndLabels();
```

Comment:
Lets duplicate [this change](https://github.com/ILIAS-eLearning/ILIAS/pull/378/commits/f978fa462735f19bb57dd18dac38949dce6fc51b) here. Consistency ftw.